src/Runner.jl

```diff
@@ -948,7 +948,7 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
         for (env_name, tool) in (
             "CC" => "$(host_bin_dir)/$(host_target)-gcc",
             "CXX" => "$(host_bin_dir)/$(host_target)-g++",
-            "DSYMUTIL" => "llvm-dsymutil",
+            "DSYMUTIL" => "dsymutil",
```

Comment:
I noticed that `llvm-dsymutil` didn't exist while investigating an issue with building an experimental GCC 11. I just thought this was my issue. Things make more sense now!